qx/qx_test.go

```diff
@@ -52,7 +52,7 @@ func TestCreateAddress(t *testing.T) {
 	s, _ := NewEntropy(32)
 	k, _ := EcNew("secp256k1", s)
 	p, _ := EcPrivateKeyToEcPublicKey(false, k)
-	a, _ := EcPubKeyToAddress("testnet", p)
+	a, _ := EcPubKeyToAddress("mixnet", p)
```

Comment:
You should not replace the original test case for `testnet`, you should add your test-case for your own.